fe/fe-core/src/main/java/com/starrocks/analysis/UpdateBackendAddressClause.java

```diff
@@ -0,0 +1,71 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
license should be this, as I was told earlier..
```
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Reply:
thanks a lot